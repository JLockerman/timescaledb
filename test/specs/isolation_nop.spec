setup{}

teardown {}

session "s1"
step ""	{}
