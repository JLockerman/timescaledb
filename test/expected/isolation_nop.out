Parsed test spec with 1 sessions

starting permutation: 
setup failed: 